hapic/__init__.py

```diff
@@ -3,7 +3,20 @@
 from hapic.data import HapicData
 from hapic.infos import __version__
 
-_hapic_default = Hapic()
+# To make a default hapic instance, must determine processor
+try:
+    from hapic.processor.marshmallow import MarshmallowProcessor
+
+    processor_class = MarshmallowProcessor
```

Comment:
`processor_class` here is `default_processor_class`, no?

Reply:
yes, i rename